### PR TITLE
Triming readme page content by referencing wiki pages

### PR DESCRIPTION
**Pull request by yue9944882**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,260 +7,26 @@
...
-Refer to the [official documentation](https://maven.apache.org/plugins/maven-deploy-plugin/usage.html) for more information.
+- [Development and Contributing](https://github.com/kubernetes-client/java/wiki/4.-Development-and-Contributing)
+- [Generate Java CRD Models](https://github.com/kubernetes-client/java/wiki/5.-Generate-Java-CRD-Model)
+- [Known Issues](https://github.com/kubernetes-client/java/wiki/6.-Known-Issues)
 
-### Maven users
+## Support
 
-Add this dependency to your project's POM:
+If you need support, start with checking whether you'r hitting known issues. If that doesn't work, please open an issue
```
Comment (brendandburns) [inline README.md:26]: you're (typo)

Comment (brendandburns) [inline README.md:26]: linkify 'open an issue' to an issue template?

https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/configuring-issue-templates-for-your-repository

File: README.md [hunk 2]
```diff
@@ -7,260 +7,26 @@
...
+to describe the cases. Additionally, before you file an issue, please search existing issues to see if your issue is
```
Comment (brendandburns) [inline README.md:27]: linkify 'existing issues' to point to the issue list?

**Comment by brendandburns** [comment]
Generally LGTM, a few comments to make it even better :)

File: README.md (hunk 1 above)
Comment (yue9944882) [reply README.md]: fixed typo

File: README.md (hunk 2 above)
Comment (yue9944882) [reply README.md]: linkified

File: README.md (hunk 1 above)
Comment (yue9944882) [reply README.md]: linkified

**Pull request merged**